docker/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.7-alpine3.9
+
+ENV ANSIBLE_LINT_VERSION=4.1.0
```

Comment:
ENV is overwritable at run time, so will not change the version of ansible-lint. 
something like

```
docker run -ti -e ANSIBLE_LINT_VERSION=4.0.0 ansible-lint
```
will have an environment variable to 4.0.0 but the version of ansible-lint of 4.1.0.
It looks like, you're looking for 'ARG' keyword, and build-args to change the image at build-time.